scalableminds/webknossos
# Fix adding sample dataset after registration bug

This PR fixes a bug that occurs when the user enters an organization name with a space in onboarding. Trying to add a sample dataset used this organization name with a space and thus created an invalid request to the server. Now directly after the user registered the received normalized organization name is used for further requests. This fixes the bug.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- maybe just checking the code should be fine, as the testing involves may steps and resetting your db.


- better test this loccaly if you want. This testing includes erasing all your data in thd db.
- Reset your database and set `insertInitialData` to false in `application.conf.
- start and open wk (the onboarding should open up).
- just floo the onboarding.
- enter "Test Organizaation" as your organization
- then register and try to add a sample dataset. The adding step should work as expected.

### Issues:
- fixes #4113

------
